<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1560</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QFrame" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1560</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="sideMenu">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8FBC94;
font-weight: bold;
color:white;
border:none;
background-color: #5442b1;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnHome">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Trang chủ</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/homeschooling.png</normaloff>:/assets/assets/icons/homeschooling.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAttendence">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Điểm danh</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/learning.png</normaloff>:/assets/assets/icons/learning.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStudent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Học sinh</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/student.png</normaloff>:/assets/assets/icons/student.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTeacher">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Giáo viên</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/teacher.png</normaloff>:/assets/assets/icons/teacher.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClass">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Lớp học</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/dictionary.png</normaloff>:/assets/assets/icons/dictionary.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSesson">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Buổi học</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/teach.png</normaloff>:/assets/assets/icons/teach.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>560</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Thống kê</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/exam.png</normaloff>:/assets/assets/icons/exam.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAccount">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>620</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Tài khoản</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/tutorial.png</normaloff>:/assets/assets/icons/tutorial.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="imgSideMenu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Hello</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubject">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Môn học</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/assets/assets/icons/books.png</normaloff>:/assets/assets/icons/books.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="contentContainer">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>1400</width>
       <height>800</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>800</height>
       </rect>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
